work/v1/types.go

```diff
@@ -138,26 +229,7 @@ type ManifestResourceMeta struct {
 // AppliedManifestResourceMeta represents the group, version, resource, name and namespace of a resource.
 // Since these resources have been created, they must have valid group, version, resource, namespace, and name.
 type AppliedManifestResourceMeta struct {
-	// Group is the API Group of the Kubernetes resource.
-	// +required
-	Group string `json:"group"`
-
-	// Version is the version of the Kubernetes resource.
-	// +required
-	Version string `json:"version"`
```

Comment:
This has a version because it is used for status reporting.  When a user identifies a resource in spec, there isn't a logical version requirement.